### ISSUE TITLE
[docs] Add missing ./securedrop-admin step in Journo Drive setup

### ISSUE THREAD

**Issue by huertanix** [freedomofpress/securedrop#4375]
## Description

On this step in the documentation: https://docs.securedrop.org/en/stable/onboarding.html#create-a-journalist-tails-usb, running `./securedrop-admin tailsconfig` fails because running `./securedrop-admin install` needs to happen first. Note that this was jotted down from a note in the worst (my) handwriting and might need a quick fact-check.

## User Research Evidence

During an admin training, an attempt to create Journo drives failed because a step was missing that was known of by the training team but not the SD admins.

**Comment by zenmonkeykstop** [comment]
This is a duplicate of #4137, which by insane coincidence I put in a PR for earlier today!

**Comment by eloquence** [comment]
Closed as duplicate, thanks for flagging David!
